Build.md

```diff
@@ -149,6 +149,14 @@ Compiling and linking TrenchBroom requires a working OpenGL installation. [This
 
 - You can replace "Release" with "Debug" if you want to create a debug build. Also change the value of the `wxWidgets_PREFIX` variable to point to your wxWidgets `build-debug` directory in that case.
 
+- Set the `TB_DEV_MODE` environment variable to `1` when launching TrenchBroom:
```

Comment:
The way this is phrased would make me ask myself if I always have to do this. Maybe rephrase it so that it is clear that this only necessary for developers.